content/docs/install/plugins.md

```diff
@@ -6,13 +6,45 @@ your data and transformations.
 
 The file itself is in a simple YAML format.
 
+We maintain a [schema](https://github.com/iterative/dvcyaml-schema) for
+`dvc.yaml` that can enable IDE syntax checks and auto-completion.
+
+## Visual Studio Code
+
+To make `dvc.lock` and `.dvc` files recognized as YAML, add the following in
+`settings.json`:
+
+```json
+"files.associations": {
+    "*.dvc": "yaml",
+    "dvc.lock": "yaml"
+}
+```
+
+Enable enhanced
+[IntelliSense](https://code.visualstudio.com/docs/editor/intellisense),
+on `dvc.yaml` files by installing the
+[YAML extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml).
+
+## PyCharm/IntelliJ
```

Comment:
```suggestion
## PyCharm & IntelliJ IDEA
```

Reply:
I that what we meant here? ☝️

Reply:
If so, the paragraph below should start with "These IDEs", probably.

Reply:
Yup, for all IDEs of Jetbrains as you wrote on `dcyaml-schema`'s README.

Reply:
Kk so
```suggestion
## JetBrain IDEs (PyCharm, Intellij IDEA, et al.)
```